lib/syntax_tree/node.rb

```diff
@@ -4175,6 +4175,18 @@ def ===(other)
     def endless?
       !bodystmt.is_a?(BodyStmt)
     end
+
+    def arity
+      @arity ||=
```

Comment:
I actually changed my mind on this. Can we skip the memoization? I think it might be surprising since this thing isn't frozen. In theory you could add params if you wanted to and then this would be off.